src/auth/oauth/oauth.types.spec.ts

```
@@ -0,0 +1,245 @@
+import {
+  BaseOAuthProvider,
+  OAuthError,
+  OAuthProviderConfig,
+  OAuthUserInfo,
+} from './oauth.types';
+
+// Mock implementation for testing BaseOAuthProvider
+class TestOAuthProvider extends BaseOAuthProvider {
+  constructor(config: OAuthProviderConfig) {
+    super(config);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)